src/entities/merch/ui/merch-card/merch-card.tsx

```diff
@@ -1,7 +1,13 @@
+'use client';
+
+import { useState } from 'react';
 import classNames from 'classnames/bind';
 import Image from 'next/image';
+import { useSwipeable } from 'react-swipeable';
 
 import { MerchProduct } from '@/entities/merch/types';
+import chevronLeft from '@/shared/assets/svg/chevron-left-solid.svg';
+import chevronRight from '@/shared/assets/svg/chevron-right-solid.svg';
```

Comment:
The Slider component uses the swiper library, can we reuse it? And maybe then we won't need new arrow icons?